l10n_it_withholding_tax_payment/i18n/l10n_it_withholding_tax_payment.pot

```diff
@@ -0,0 +1,268 @@
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+msgid "Tot"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.ui.menu,name:l10n_it_withholding_tax_payment.menu_withholding_tax_move_payment
+msgid "WT Move Payments"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__wt_move_ids
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.wizard_wt_move_payment_create_view
+msgid "WT Moves"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__amount
+msgid "WT amount"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model,name:l10n_it_withholding_tax_payment.model_wizard_wt_move_payment_create
+msgid "WT wizard create move payment"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: code:addons/l10n_it_withholding_tax_payment/models/withholding_tax.py:77
+#, python-format
+msgid "Warning! Datas required for account move creation: payment date, journal, account"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: code:addons/l10n_it_withholding_tax_payment/models/withholding_tax.py:33
+#, python-format
+msgid "Warning! Withholding Tax moves in a payment: {}"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_tree
+msgid "Withholding Move Payment"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model,name:l10n_it_withholding_tax_payment.model_withholding_tax_move
+msgid "Withholding Tax Move"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model,name:l10n_it_withholding_tax_payment.model_withholding_tax_move_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.view_withholding_move_payment_form
+msgid "Withholding Tax Move Payment"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: code:addons/l10n_it_withholding_tax_payment/models/withholding_tax.py:110
+#: code:addons/l10n_it_withholding_tax_payment/models/withholding_tax.py:118
+#: code:addons/l10n_it_withholding_tax_payment/wizard/create_move_payment.py:29
+#, python-format
+msgid "Withholding Tax Payment"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: code:addons/l10n_it_withholding_tax_payment/models/withholding_tax.py:91
+#, python-format
+msgid "Withholding Tax Payment %s"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.actions.act_window,name:l10n_it_withholding_tax_payment.action_withholding_tax_move_payment
+msgid "Withhoulding Tax Move Payment"
```

Comment:
Vedi sopra, inoltre
s/Withhoulding/Withholding

Reply:
👌

Reply:
Idem per le maiuscole,in tutti i punti in cui compare la stringa.

Reply:
👌